internal/fs/wrappers/error_mapping.go

```diff
@@ -59,16 +60,28 @@ func errno(err error) error {
 	}
 
 	// Cannot authenticate or Permission denied.
-	// The control client API returns an RPC error code instead of googleapi code.
-	if strings.Contains(err.Error(), "oauth2: cannot fetch token") || strings.Contains(err.Error(), "PermissionDenied") {
+	if strings.Contains(err.Error(), "oauth2: cannot fetch token") {
 		return syscall.EACCES
 	}
 
-	// Translate API errors into an em errno
-	var apiErr *googleapi.Error
+	// The control client API returns an RPC error code instead of googleapi code.
+	var apiErr *apierror.APIError
```

Comment:
apiError has a method FromError(). can you use that to write unit tests?

Reply:
FromError parses a Status error or a googleapi.Error. RPC errors require a Status error. However, Status has private attributes that cannot be assigned from outside its package. This prevents us from creating a Status error, and thus we cannot use FromError to create an apiError.

Reply:
Done